addon/plugins/table/index.ts

```diff
@@ -29,14 +29,27 @@ export class TableView extends PluginTableView {
     public cellMinWidth: number,
   ) {
     super(node, cellMinWidth);
-    this.addClasses(node);
+    this.addAttrs(node);
   }
 
-  private addClasses(node: Node): void {
-    const nodeClasses = node.attrs.class as string | undefined;
+  private addAttrs(node: Node): void {
+    const {
+      class: nodeClasses,
+      style,
+      ...attrs
+    } = node.attrs as Record<string, unknown>;
     if (typeof nodeClasses === 'string') {
       this.table.classList.add(...nodeClasses.split(' '));
     }
+    if (typeof style === 'string') {
+      this.table.style.cssText = `${this.table.style.cssText} ${style}`;
+    }
```

Comment:
It seems we now both define the border-styles in inline css **and** external css for the dynamic table view. We can probably remove the external css on tables **or** not apply the border inline-css to the dynamic node-view.